src/fdw/base.rs

```diff
@@ -246,3 +255,29 @@ pub enum BaseFdwError {
     #[error(transparent)]
     Options(#[from] OptionsError),
 }
+
+struct DuckDbFormatter {}
+
+impl CellFormatter for DuckDbFormatter {
+    fn fmt_cell(&mut self, cell: &Cell) -> String {
+        match cell {
+            Cell::Bytea(v) => {
+                let byte_u8 = unsafe { varlena_to_byte_slice(*v) };
+                let hex = byte_u8
+                    .iter()
+                    .map(|b| format!(r#"\x{:02X}"#, b))
+                    .collect::<Vec<String>>()
+                    .join("");
```

Comment:
Here too.

Instead of allocated Vec and then join. Can we try to use `.intersperse(" AND ".to_string()) .collect::<String>();`?

Reply:
see above.